Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -1443,8 +1828,17 @@ def main():
             demisto.setLastRun(next_run)
             demisto.incidents(incidents)
 
+        elif demisto.command() == 'get-modified-remote-data':
+            return_results(get_modified_remote_data_command(client, args))
+        elif demisto.command() == 'get-remote-data':
+            return_results(get_remote_data_command(client, args))
+        elif demisto.command() == 'get-mapping-fields':
+            return_results(get_mapping_fields_command())
+        elif demisto.command() == 'update-remote-system':
+            return_results(update_remote_system_command(client, args))
```

Comment:
why you didn't add this commands to `commands` and so the `elif` won't be necessary and line 1841 will be tnough?

Reply:
Because the function get_mapping_fields_command does not take arguments, and it will have to be separate, I thought to already attach to it all the commands that belong to mirroring?
WYT?

Reply:
I think it's better to add a comment in the `commands`, something like this:
`# mirroring commands`
`'get-modified-remote-data': get_modified_remote_data_command,`
and leave only the `elif` for `'get-mapping-fields'`

Reply:
OK